dylibbundler: update to 1.0.0

Lord-Kamina opened the pull request:
#### Description

<!-- Note: it is best to make pull requests from a branch rather than from master -->

###### Type(s)
<!-- update (title contains ": U(u)pdate to"), submission (new Portfile) and CVE Identifiers are auto-detected, replace [ ] with [x] to select -->

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
<!-- Triple-click and copy the next line and paste it into your shell. It will copy your OS and Xcode version to the clipboard. Paste it here replacing this section.
sh -c 'printf "%s\n" "macOS `sw_vers -productVersion` `sw_vers -buildVersion` `uname -m`" "`xcodebuild -version|awk '\''NR==1{x=$0}END{print x" "$NF}'\''`"'|tee /dev/tty|pbcopy
-->
macOS x.y
Xcode x.y

###### Verification <!-- (delete not applicable items) -->
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?

<!-- Use "skip notification" (surrounded with []) to avoid notifying maintainers -->

Just one note, I added the `configure.cxx_standard` and it did nothing, not sure if it's because the port doesn't use configure or what, but that's the reason I *also* added the `-std=c++11` flag like that in the `CXXFLAGS`.

ryandesign commented:
Thanks for letting me know version 1.0.0 was available.

> Just one note, I added the `configure.cxx_standard` and it did nothing,

`compiler.cxx_standard` doesn't do nothing: it asks MacPorts to select a compiler that supports that language standard.

> that's the reason I _also_ added the `-std=c++11` flag like that in the `CXXFLAGS`.

The Makefile already adds this flag, we were just overriding the Makefile. I've fixed it so that we no longer override the Makefile for this flag.

Lord-Kamina commented:
> Thanks for letting me know version 1.0.0 was available.
> 
> > Just one note, I added the `configure.cxx_standard` and it did nothing,
> 
> `compiler.cxx_standard` doesn't do nothing: it asks MacPorts to select a compiler that supports that language standard.
> 
> > that's the reason I _also_ added the `-std=c++11` flag like that in the `CXXFLAGS`.
> 
> The Makefile already adds this flag, we were just overriding the Makefile. I've fixed it so that we no longer override the Makefile for this flag.

Ah, I see.
